components/badge/badge-sup.component.ts

```diff
@@ -67,6 +68,7 @@ export class NzBadgeSupComponent implements OnInit, OnChanges {
   @Input() disableAnimation = false;
   @Input() nzCount?: number | TemplateRef<NzSafeAny>;
   @Input() noAnimation = false;
+  @Input() nzSize: 'default' | 'small' = 'default';
```

Comment:
Please add some tests for `nzSize`, and use `NzSizeDSType` instead of string :)

Reply:
Ok, I will correct it in a few days

Reply:
done